### PR TITLE
Karlgrin/issue #58

### PR DESCRIPTION
**Pull request by karlgrin**

### CODE REVIEW

**Comment by karlgrin** [comment]
Maybe #75 is still outstanding. Need to check this more thoroughly.

**Comment by weinrank** [comment]
PR isn't mergeable, please resolve conflicts before merge

File: neat_log.c [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 #ifdef NEAT_LOG
 
-uint8_t neat_log_level = NEAT_LOG_INFO;
+uint8_t neat_log_level = NEAT_LOG_DEBUG;
```
Comment (weinrank) [inline neat_log.c]: NEAT_LOG_INFO should be kept

**Comment by weinrank** [comment]
I tried to integrate your changes into the current master branch which caused some problems especially with use-after-free errors.

I suggest that you fix the issues in two steps within a branch of the current master to get a clean solution.

**Comment by karlgrin** [comment]
BTW did HE between SCTP and TCP work (Issue #75)?

**Comment by weinrank** [comment]
Regardless of errors: Yes!

http://buildbot.nplab.de:28010/waterfall?branch=weinrank/he_fix

There you see three runs - the second one corresponds to your changes. 
FreeBSD-ARM fails and also Ubuntu with IPv6 but it's better than before.

**Pull request closed without merging**